redcap_bridge/utils.py

```diff
@@ -46,4 +49,34 @@ def get_repo_state(path):
         commit_hash = ''
         clean = False
 
-    return commit_hash, clean
\ No newline at end of file
+    return commit_hash, clean
+
+
+def compressed_record(csv_file, compressed_file=None):
+    custom_csv = pd.read_csv(csv_file)
+    df = pd.DataFrame(custom_csv)
+    custom_df = df.filter(regex='.___.')
+    for column in custom_df:
+        if 1 in custom_df[column].values:
+            style_moda = re.search('___(.+?)', column).group(1)
+            custom_df[column] = style_moda
+        else:
+            del custom_df[column]
```

Comment:
I think `del` is not the ideal function to remove a column from a dataframe. `Del` is usually used in cases where you need to remove an object from memory when the python garbage collection is not doing the job quickly enough. I guess dataframes should have a method to `pop` or `remove` a column. Did you encounter something like this already?

Reply:
Okay, I didn't know del had this usage. So I modified it using .pop()